integration/apps/rails-five/config/puma.rb

```diff
@@ -0,0 +1,24 @@
+require 'datadog/demo_env'
+
+Datadog::DemoEnv.print_env('Puma master environment')
+
+workers 1
+threads 2, Integer(ENV['RAILS_MAX_THREADS'] || 24)
```

Comment:
Should we enable a few more workers/threads, just to cause a bit more stress during testing?

Reply:
We definitely have that option via the ENV for the container. Probably better suited doing this in our performance testing environment.

Reply:
👍  I also doublechecked and with `workers 1` puma already starts two processes (a coordinator and a worker) -- I was thinking that we'd need `workers 2` for that.